lib/rubocop/cop/lint/unmodified_reduce_accumulator.rb

```diff
@@ -68,7 +67,10 @@ class UnmodifiedReduceAccumulator < Base
         MSG_INDEX = 'Do not return an element of the accumulator in `%<method>s`.'
 
         def_node_matcher :reduce_with_block?, <<~PATTERN
-          (block (send _recv {:reduce :inject} ...) args ...)
+          {
+            (block (send _recv {:reduce :inject} ...) args ...)
+            (numblock (send _recv {:reduce :inject} ...) ...)
+          }
```

Comment:
Wouldn't `({block numblock} (send _recv {:reduce :inject} ...) ...)` work?

Reply:
Yeah but I thought it would be better to only consider blocks that explicitly have args (which `numblock`s do not)?